apps/explorer/templates/explorer/pixelset_detail.html

```diff
@@ -127,45 +127,47 @@ <h4>{% trans "Subset selection" %}</h4>
       <p>
         {% blocktrans %}
           Copy/paste a list of omics units to select a subset of pixels in this
-          pixel set. You'll be able to export them further.
+          pixel set and export them.
         {% endblocktrans %}
       </p>
 
       <div class="form-error">
-        {{ export_form.non_field_errors }}
+        {{ form.non_field_errors }}
       </div>
 
-      <form action="{{ pixelset.get_export_pixels_url }}" method="post" id="pixel-selection-form">
+      <form action="{{ pixelset.get_absolute_url }}" method="post" id="pixel-selection-form">
         {% csrf_token %}
 
-        <label>
-          Omics units
-          <textarea
-            name="omics_units"
-            rows="15"
-            placeholder="CAGL0F02695g, CAGL0D06336g, CAGL0G06666g"
-          ></textarea>
-        </label>
-        <p class="help-text">
-          {% blocktrans %}
-            Type omics units separated by a coma, a carriage return or a space
-          {% endblocktrans %}
-        </p>
-        <button
-          type="submit"
-          class="button"
-        >
-          <i class="fa fa-download" aria-hidden="true"></i>
-          {% trans "Download a CSV file with the selected Pixels" %}
-        </button>
+        {{ form.as_p }}
```

Comment:
Why using `as_p`?

Reply:
You should use the foundation form template:

```
{% include "foundation/form.html" %}
```

Reply:
`as_p` because otherwise you can a `<br>` after `textarea`